javascript/example/DRACOLoader.js

```diff
@@ -177,10 +177,12 @@ THREE.DRACOLoader.prototype = {
             geometryBuffer.vertices[i + 1] = posAttributeData.GetValue(i + 1);
             geometryBuffer.vertices[i + 2] = posAttributeData.GetValue(i + 2);
             // Add color.
+            // 03/05/2017, mackuntu@: ThreeJS vertex colors need to be
```

Comment:
Can you change this to not have the date and username. We don't usually put usernames in the comments unless it is a TODO.

If you want, you can add your username to the AUTHORS file.